packages/core/strapi/lib/commands/transfer/import.js

```diff
@@ -22,9 +22,25 @@ const {
 const { exitWith } = require('../utils/helpers');
 
 /**
- * @typedef {import('@strapi/data-transfer').ILocalFileSourceProviderOptions} ILocalFileSourceProviderOptions
+ * @typedef {import('@strapi/data-transfer/src/file/providers').ILocalFileSourceProviderOptions} ILocalFileSourceProviderOptions
```

Comment:
Just something I noticed and will investigate later.

Not sure why it's necessary, but without the path the type isn't actually loaded. if it's a bug with how we export the types we can fix in a later PR.